packages/bpk-docs/src/components/DocsPageWrapper/DocsPageWrapper.js

```diff
@@ -106,7 +92,7 @@ const DocsPageWrapper = props => {
     history,
     location,
   } = props;
-  const path = getBasePath(match.url);
+  const path = match.url;
```

Comment:
Isn't this confusing? If it's a path why are we setting it to a URL? path != url

Reply:
it's actually the path, e.g. `/components/badge` and not the URL 🤷‍♂️ . `match.url` is from react router so not sure we can do something about it.